client/webpack.config.js

```diff
@@ -42,7 +50,7 @@ const CONFIG = {
   module: {
     rules: [
       {
-        test: /\.js$/,
+        test: /\.js|.jsx$/,
```

Comment:
this just makes it possible to use the .jsx extension in the React component files. this can help with syntax plugins.